test/Interop/Cxx/operators/member-inline-silgen.swift

```diff
@@ -32,3 +32,16 @@ public func call(_ wrapper: inout AddressOnlyIntWrapper) -> Int32 { wrapper() }
 // CHECK: end_access [[SELFACCESS]] : $*AddressOnlyIntWrapper
 
 // CHECK: sil [clang AddressOnlyIntWrapper.callAsFunction] [[NAME]] : $@convention(c) (@inout AddressOnlyIntWrapper) -> Int32
+
+public func index(_ wrapper: inout ReadOnlyIntArray, _ arg: Int32) -> Int32 { wrapper[arg] }
+
+// CHECK: bb0([[SELF:%.*]] : $*ReadOnlyIntArray, [[RHS:%.*]] : $Int32):
+// CHECK: [[SELFACCESS:%.*]] = begin_access [modify] [static] [[SELF]] : $*ReadOnlyIntArray
+// CHECK: [[SELFACCESS2:%.*]] = begin_access [modify] [static] [[SELFACCESS]] : $*ReadOnlyIntArray
+// CHECK: [[OP:%.*]] = function_ref [[NAME:@(_ZNK16ReadOnlyIntArrayixEi|\?\?0ReadOnlyIntArray@@QEAA@XZ)]] : $@convention(c) (@inout ReadOnlyIntArray, Int32) -> UnsafePointer<Int32>
+// CHECK: [[PTR:%.*]] = apply [[OP]]([[SELFACCESS2]], [[RHS]]) : $@convention(c) (@inout ReadOnlyIntArray, Int32) -> UnsafePointer<Int32>
+// CHECK: end_access [[SELFACCESS2]] : $*ReadOnlyIntArray
+// CHECK: struct_extract [[PTR]] : $UnsafePointer<Int32>, #UnsafePointer._rawValue
+// CHECK: end_access [[SELFACCESS]] : $*ReadOnlyIntArray
+
```

Comment:
Please also check `_operatorSubscript` and its body here and in the IRGen test. We want to make sure we check the place where we actually call the C++ operator.

Reply:
I think you meant subscript getters/setters, and in that case, fixed.